lib/graphql/transfer.ts

```diff
@@ -261,7 +261,7 @@ export class Transfer extends IterableModel<
   /**
    * Update a standing order
    *
-   * @param transfer  transfer data including at least id and type. For Timed Orders and Sepa Transfers, only category and userSelectedBooking date can be updated 
+   * @param transfer  transfer data including at least id and type. For Timed Orders and Sepa Transfers, only category and userSelectedBooking date can be updated
```

Comment:
Totally unrelated but my editor kept removing this, so I added a separate commit just for this change